src/main/java/Task.java

```diff
@@ -0,0 +1,36 @@
+public class Task {
+    protected final String taskDescription;
+    protected boolean isDone;
+    protected static int numOfTasks = 0;
+
+    public Task(String taskDescription) {
+        this.taskDescription = taskDescription;
+        this.isDone = false;
+        numOfTasks += 1;
+    }
+
+    public static int getNumOfTasks() {
+        return numOfTasks;
+    }
+
+    public String getTaskDescription() {
+        return taskDescription;
+    }
+
+    public void setDone() {
+        this.isDone = true;
+    }
+
+    public void setUndone() {
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "[X] " : "[ ] ");
```

Comment:
Perhaps consider using explicit if-else statements as ternary operators may not be obvious to some